pxr/usdImaging/lib/usdImagingGL/drawModeAdapter.cpp

```diff
@@ -1092,11 +1092,23 @@ UsdImagingGLDrawModeAdapter::_ComputeExtent(UsdPrim const& prim) const
 {
     HD_TRACE_FUNCTION();
     HF_MALLOC_TAG_FUNCTION();
-
-    TfTokenVector purposes = { UsdGeomTokens->default_, UsdGeomTokens->proxy,
-                               UsdGeomTokens->render };
-    UsdGeomBBoxCache bboxCache(UsdTimeCode::EarliestTime(), purposes, true);
-    return bboxCache.ComputeUntransformedBound(prim).ComputeAlignedBox();
+    if (prim.IsLoaded()) {
+        TfTokenVector purposes = { UsdGeomTokens->default_,
+            UsdGeomTokens->proxy, UsdGeomTokens->render };
+        UsdGeomBBoxCache bboxCache(
+            UsdTimeCode::EarliestTime(), purposes, true);
+        return bboxCache.ComputeUntransformedBound(prim).ComputeAlignedBox();
+    } else {
+        GfRange3d extent;
+        if (auto attr = UsdGeomModelAPI(prim).GetExtentsHintAttr()) {
+            VtVec3fArray extentHints;
+            if (attr.Get(&extentHints, UsdTimeCode::EarliestTime()) &&
```

Comment:
Do we need a null check for attr here?

Reply:
I think the`if (auto attr = ...)` code does a null pointer check already?

Reply:
Yeah, it returns an object not a pointer, but the object's validity _is_ guarded by the check of the auto local.